core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly

```diff
@@ -282,14 +195,6 @@ THE SOFTWARE.
                             <f:submit value="${%Create account}"/>
                         </div>
                         <input type="hidden" name="${h.getCrumbRequestField()}" value="${h.getCrumb(request)}"/>
-                        <j:if test="${data.errorField != null}">
-                            <script type="text/javascript">
-                                const errorField = document.getElementById('${data.errorField}');
-                                if(errorField &amp;&amp; errorField.focus) {
-                                    errorField.focus();
-                                }
-                            </script>
-                        </j:if>
```

Comment:
Added during development of #3380 but the supporting code elsewhere was removed before it was merged, so this was always false.